keras_hub/src/models/phi4/phi4_layernorm.py

```diff
@@ -0,0 +1,35 @@
+import keras
+from keras import ops
+
+
+# TODO: Deprecate this in favor of
+# `keras.layers.LayerNormalization(rms_scaling=True)` once Keras 2 support is
+# removed.
```

Comment:
We don't have keras 2 support now, either update the code or remove/update this comment.